app/src/androidTest/java/org/simple/clinic/RegisterAppointmentRule.kt

```diff
@@ -0,0 +1,48 @@
+package org.simple.clinic
```

Comment:
This rule is currently being used only in `CommunicationSyncAndroidTest`, but I couldn't figure out how to elegantly register an appointment before running the test since it actually extends `BaseSyncCoordinatorAndroidTest`. So I ended up creating a rule for it.

Reply:
Yeah, sure. I wasn't really stuck on this; Didn't spend too much time on trying to work around this and went ahead with a rule.

Using a `@Before` is interesting, but I am not sure what exactly will happen if we added an `@Before` in the superclass as well. I'll try it out.

Reply:
Oh, nvm. There's already a fixture in this test. I'll use this to set up the appointment.